crates/core/src/db/relational_db.rs

```diff
@@ -256,16 +272,29 @@ impl RelationalDB {
     /// state. See also [`Self::with_auto_commit`].
     #[tracing::instrument(skip_all)]
     pub fn begin_tx(&self) -> MutTx {
-        log::trace!("BEGIN TX");
+        log::trace!("BEGIN Mut TX");
```

Comment:
```suggestion
        log::trace!("BEGIN MUT TX");
```

Just for consistency